python/ray/train/_internal/storage.py

```diff
@@ -507,7 +506,11 @@ def _check_validation_file(self):
             raise RuntimeError(
                 f"Unable to set up cluster storage with the following settings:\n{self}"
```

Comment:
nit: We probably don't want to expose the whole StorageContext if it's an internal concept, but I think it's okay for now. We can revisit if it's causing confusion in the future.

Reply:
oh, what about `("local", "/path/on/fs")` instead then?